include/swift/Serialization/ModuleFile.h

```diff
@@ -562,8 +562,7 @@ class ModuleFile
   ///
   /// If the record at the cursor is not a generic param list, returns null
   /// without moving the cursor.
-  GenericParamList *maybeReadGenericParams(DeclContext *DC,
-                                     GenericParamList *outerParams = nullptr);
+  GenericParamList *maybeReadGenericParams(DeclContext *DC);
```

Comment:
Could you bump the module file version? Even though you didn't change the code, the way we represent generic parameter lists is so different that there's zero compatibility between module files before/after.